### PR TITLE
remove connections from b (config) and some legacy pulseaudio support

### PR DESCRIPTION
**Pull request by Fl1tzi**

### CODE REVIEW

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            if [[ $file == *.py ]]; then
+              echo
+              echo "-------[ $file ]-------"
+              echo
+          
+              pylint \
+              --load-plugins perflint \
+              --disable W8205,W8201,W0212,W0613,R0903,W0614,C0111,C0301,C0103,F0401,W0402,R0914,C0321 \
+              --output-format colorized \
+              --max-line-length 80 \
```
Comment (theRealCarneiro) [inline .github/workflows/pylint.yml:71]: pls no max line length

**Review by theRealCarneiro — COMMENTED** [review]
80 characters limit is not really required anymore

**Comment by Fl1tzi** [comment]
> 80 characters limit is not really required anymore

Why?

File: .github/workflows/pylint.yml (hunk 1 above)
Comment (Fl1tzi) [reply .github/workflows/pylint.yml]: > pls no max line length

removed

File: .github/workflows/pylint.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      run: |
+          for file in ${{ steps.changed-files-else.outputs.all_changed_files }}; do
+            if [[ $file == *.py ]]; then
+              echo
+              echo "-------[ $file ]-------"
+              echo
+          
+              pylint \
+              --load-plugins perflint \
+              --disable W8205,W8201,W0212,W0613,R0903,W0614,C0111,C0301,C0103,F0401,W0402,R0914,C0321 \
```
Comment (theRealCarneiro) [inline .github/workflows/pylint.yml:69]: I use ignore = E402, E128, E116, E701, E501, E503, E225, W504 in my pycodestyle, can you check if they're there?

**Pull request merged**